README.md

```diff
@@ -40,14 +40,110 @@ See [LevelDB Persistence](https://github.com/yjs/y-leveldb) for more info.
 PORT=1234 YPERSISTENCE=./dbDir node ./node_modules/y-websocket/bin/server.js
 ```
 
-### Scaling
+#### Start a secure websocket server
+
+You can also use the secure `wss://` protocol, which like `https://`, uses SSL/TLS.  For this, you first need a certificate.  If your server is  using `https://`, you will already have a certificate.  If not, you can obtain one for free from [certbot](https://certbot.eff.org/).  You should then provide the directory in which the certifcate and private key are to be found:
+
+```sh
+PORT=1234 YPERSISTENCE=./dbDir CERTDIR=/etc/letsencrypt/live/example.com node ./node_modules/y-websocket/bin/server.js
+```
+
+You would then connect using the `wss://` protocol:
+
+```js
+const wsProvider = new WebsocketProvider('wss://localhost:1234', 'my-roomname', doc)
+```
+
+#### Providing a service
```

Comment:
There are many different methods to run a nodejs executable. I don't think it should be part of the y-websocket docs. 

It would be nice to find a website that explains different websites to run a nodejs service. I wouldn't just want to recommend a single service. systemd is really hard to maintain for novices. And people who are familiar with it are able to find out how to run a node application. The below recommendation is not very generic and points to a specific nodejs version and it also runs the service as `root`, which you should avoid.

Instead, you could run your application with pm2, or something similar.